src/components/ApartmentApplicantList/index.js

```diff
@@ -1,25 +1,47 @@
 import React, { Component } from 'react';
 import 'date-fns';
-import { Grid, Card, CardHeader, CardContent, List } from '@material-ui/core';
+import { Grid, Card, CardHeader, CardContent, List, Button, Typography } from '@material-ui/core';
 import GroupAddIcon from '@material-ui/icons/GroupAdd';
+import ErrorIcon from '@material-ui/icons/Error';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import { gordonColors } from '../../theme';
 import GordonPeopleSearch from '../Header/components/PeopleSearch';
+import GordonLoader from '../../components/Loader';
 import ApplicantListItem from './components/ApplicantListItem';
 import '../../views/ApartmentApp/apartmentApp.css';
 import '../../views/PeopleSearch/components/PeopleSearchResult/peopleSearchResult.css';
+const MIN_NUM_APPLICANTS = 3;
```

Comment:
I think there can actually be as few as 2 applicants in the case of the Village

Reply:
I have commented this out in the next version up, but I realized there should not be a minimum check for save, since we allow saving a partially completed state, but we will need to have this check with the submit button